### DISCUSSION TITLE
第1节课笔记(2班)

### DISCUSSION THREAD

**Discussion by MING-ZCH** [InternLM/Tutorial#50] [笔记]
## 2班第一节课笔记交这里哟~
* 链接形式提交
* 也记得在信息收集文档对应作业处填写链接

**Comment by RyanWu31** [comment]
mark

**Comment by jinxianwei** [comment]
[大模型实战营01](https://blog.csdn.net/jinxianwei1999/article/details/135372517?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135372517%22%2C%22source%22%3A%22jinxianwei1999%22%7D)

**Comment by linmiaoyan** [comment]
[林水火](https://www.bilibili.com/read/cv28999771/?spm_id_from=333.999.0.0)

**Comment by Mr-Vincenyang** [comment]
[书生·浦语大模型全链路开源体系[大模型实战营01]](https://zhuanlan.zhihu.com/p/675930094)阿文

**Comment by kyrie2to11** [comment]
[InterLM L-1: 书生·浦语大模型全链路开源开放体系](https://zhuanlan.zhihu.com/p/675925793) 打卡~

**Comment by DarthThomas** [comment]
[第一课笔记](https://share-tianhe.notion.site/fd7a200cc27a483db0d1fe58f6b5783d?pvs=4)

**Comment by wang-kangkang** [comment]
[第一课笔记](https://zhuanlan.zhihu.com/p/675935582?)

**Reply by MING-ZCH** [reply]
同学，请将链接填写在群公告信息收集表名字对应处哦~
信息收集表没找你lol

**Reply by MING-ZCH** [reply]
已帮你填写，打扰啦~

**Comment by xrxu** [comment]
https://github.com/xrxu/Docu?tab=readme-ov-file

**Comment by kishiokon** [comment]
[书生·浦语大模型全链路开源开放体系](https://www.bilibili.com/read/cv29004021/?jump_opus=1)

**Comment by dcuplover** [comment]
[生·浦语大模型全链路开源体系](https://www.yuque.com/u174176/new86d/vhxunlbkreanpqay?singleDoc#%20%E3%80%8A%E3%80%90%E5%AD%A6%E4%B9%A0%E3%80%91%E7%AC%AC%E4%B8%80%E8%AF%BE%20%E4%B9%A6%E7%94%9F%C2%B7%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%85%A8%E9%93%BE%E8%B7%AF%E5%BC%80%E6%BA%90%E4%BD%93%E7%B3%BB%E3%80%8B)

**Comment by hxdsdu** [comment]
第一课笔记https://github.com/hxdsdu/shusheng/blob/main/README.md

**Reply by MING-ZCH** [reply]
同学，请将个人信息填写到群公告信息收集表中哦~
你的算力平台用户ID未填写~

**Comment by ZixinxinWang** [comment]
https://blog.csdn.net/Wzxdecsdn/article/details/135384534?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135384534%22%2C%22source%22%3A%22Wzxdecsdn%22%7D

**Comment by yanxiaoqi932** [comment]
https://wspqhnu603.feishu.cn/docx/OlcsdceDiom5ocxPE6UcWq2en4f?from=from_copylink

**Comment by CoderJackZhu** [comment]
[知乎](https://zhuanlan.zhihu.com/p/676064984)

**Comment by YanxingLiu** [comment]
https://s5eixc6gba.feishu.cn/docx/RyQbdHN9yoqNcSxFyzucxpLwnTu?from=from_copylink

**Comment by xulinui** [comment]
https://h90ag9106t.feishu.cn/docx/Dl6gdmiydohZp5x95RDc89rynnd?from=from_copylink

**Comment by ljqiff** [comment]
[http://t.csdnimg.cn/lKyKt](书生·浦语大模型全链路开源体系)

**Comment by enkilee** [comment]
mark

**Comment by ljqiff** [comment]
已收到你的邮件。

**Comment by phdbrianlee** [comment]
https://www.jianshu.com/p/c7b150f13efb

打卡

**Comment by tcexeexe** [comment]
[【书生·浦语实战营】第一课：书生技术生态系统概览](https://blog.csdn.net/tc_exe/article/details/135398254?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135398254%22%2C%22source%22%3A%22tc_exe%22%7D)
打卡

**Comment by majestichou** [comment]
> ## 2班第一节课笔记交这里哟~
> * 链接形式提交

笔记链接为：https://github.com/InternLM/tutorial/discussions/50

**Comment by majestichou** [comment]
现阶段使用Opencompass对开源大模型CodeLlama-Instruct-13B进行了测评。测评发现一个很神奇的现象：Codellama assessment score with meta prompt is half lower than codellama assessment score without meta prompt！使用llama2的提示词模板格式撰写了meta prompt，测评CodeLlama-Instruct-13B在Humaneval数据集上的表现，得分是12.8；将meta prompt置空，测评CodeLlama-Instruct-13B在Humaneval数据集上的表现，得分是28.05。已经向Opencompass github仓库提交Issue，详情见https://github.com/open-compass/opencompass/issues/768

**Comment by qihong2019** [comment]
[书生·浦语大模型全链路开源体系](https://www.processon.com/view/link/65975cb91a610d40bb8389f7)

**Comment by enkilee** [comment]
[书生·浦语大模型全链路开源体系 lesson 1](https://blog.csdn.net/luhputu520/article/details/135407067)

**Comment by AstrumPush** [comment]
第一课笔记：https://www.yuque.com/xing_chen/ydwipg/gk7esgosd6bszvmh?singleDoc# 《0103 书生·浦语大模型全链路开源体系》

**Comment by zyy0831** [comment]
第一课笔记：https://blog.csdn.net/sinat_41838539/article/details/135410511?spm=1001.2014.3001.5502

**Comment by KylinCarol** [comment]
笔记①：https://blog.csdn.net/m0_65412000/article/details/135415305?spm=1001.2014.3001.5501

**Comment by heiqiyihu-sp** [comment]
第一课笔记
[Uploading 全链条开源开放体系.pdf…]()

**Comment by hongxiaoy** [comment]
# **书生·浦语大模型全链路开源体系 Lec 1**

1. 大模型成为热门关键词
2. 大模型成为发展通用人工智能的重要途径
    1. 专用模型：针对特定任务，一个模型解决一个问题
    2. 通用大模型：一个模型应对多种任务、多种模态
3. 书生·浦语大模型
4. 书生·浦语大模型系列
    1. 轻量级：InternLM-7B
    2. 中量级：InternLM-20B
    3. 重量级：InternLM-123B
5. 书生·浦语20B开源大模型性能
6. 从模型到应用：
    1. 书生·浦语 —> 智能客服、个人住手、行业应用
7. 全链条开源开放体系
    1. 数据：书生·万卷1.0
        1. 文本数据
        2. 图像-文本数据集
        3. 视频数据
    2. 预训练
    3. 微调：XTuner
        1. 增量续训：学习新知识
        2. 有监督微调：学会理解和遵循各种指令
    4. 部署：LMDeploy
    5. 评测：OpenCompass
        1. 学科
        2. 语言
        3. 知识
        4. 理解
        5. 推理
        6. 安全
    6. 应用：Lagent、AgentLego

**Reply by MING-ZCH** [reply]
同学，笔记不错~
请填写群公告信息收集表格，方便我们对后续打分和评选优秀学员哦

**Comment by pranerd** [comment]
[第一课笔记csdn](https://blog.csdn.net/gaxzj/article/details/135415566?spm=1001.2014.3001.5501)

**Comment by bjzhb666** [comment]
书生·浦语大模型全链路开源体系：自研大模型基础介绍

**Reply by MING-ZCH** [reply]
bro链接没填上哦~

**Comment by WanpengXu** [comment]
[书生·浦语大模型实战营-第一节](https://b07ofnm9xj7.feishu.cn/wiki/B7XawUAO5i6r47kwZsMc9MaLnfh)

**Comment by PrinceRay7** [comment]
[第一节课笔记](https://blog.csdn.net/Prince_Ray/article/details/135417010?spm=1001.2014.3001.5502)

**Comment by yangff0016** [comment]
https://blog.csdn.net/weixin_42837310/article/details/135417482?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135417482%22%2C%22source%22%3A%22weixin_42837310%22%7D

**Comment by zihaoworks** [comment]
https://docs.qq.com/doc/DTFV6SW9qaU1iY1pM

**Comment by songyuc** [comment]
[http://t.csdnimg.cn/Leif6](http://t.csdnimg.cn/Leif6)

**Comment by RyanWu31** [comment]


**Reply by MING-ZCH** [reply]
好的，已收到~打扰啦

**Reply by RyanWu31** [reply]
https://github.com/RyanWu31/tutorial_notebook/blob/main/README.md

**Reply by MING-ZCH** [reply]
请将链接提交至第二节课作业处~
并在群公告信息收集表填写链接~
